src/components/Blockquote.stories.tsx

```diff
@@ -1,42 +1,29 @@
-import 'braid-design-system/reset';
 import 'loki/configure-react';
 
 import { List, Text } from 'braid-design-system';
-import React, { type ComponentProps } from 'react';
-
-import {
-  type BraidArgs,
-  defaultArgTypes,
-  defaultArgs,
-} from '../storybook/controls';
-import { BraidStorybookProvider, withRouter } from '../storybook/decorators';
+import React from 'react';
+import type { StoryObj } from 'sku/@storybook/react';
 
 import { Blockquote as Component } from './Blockquote';
 
 export default {
-  args: {
-    braidThemeName: defaultArgs.braidThemeName,
-    size: defaultArgs.size,
-  },
-  argTypes: {
-    braidThemeName: defaultArgTypes.braidThemeName,
-    size: defaultArgTypes.size,
-  },
-  component: Component,
-  decorators: [withRouter],
   title: 'Standalone/Blockquote',
+  component: Component,
 };
```

Comment:
```suggestion
} satifies Meta<typeof Component>;
```
https://github.com/SEEK-Jobs/ie-ad-selection-panel/blob/master/src/panels/example/Example.stories.tsx#L11